maven-plugin/src/main/resources/help.yaml

```diff
@@ -24,9 +24,21 @@ help:
         desc: "Add demo data when setting up a platform server."
       - name: "Dfile"
         desc: "Path to installation.properties file for batch setup."
+      - name: "DdockerHost"
+        desc: "Docker host address"
+      - name: "DdbReset"
+        desc: "If 'true' reset the database"
+      - name: "DjavaHome"
+        desc: "Path to JDK Version"
```

Comment:
Path to JDK version or home?

Reply:
I used this: https://github.com/openmrs/openmrs-sdk/blob/12d3ba529b963ab1e8b81e55ce195e883d8facb1/maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java#L109-L114

I think it should be JAVA_HOME so I changed it in the new commit.